packages/mux-player/src/styles.css

```diff
@@ -30,6 +30,10 @@ media-control-bar [role="switch"] {
   background-color: var(--media-control-background, rgba(20, 20, 30, 0.7));
 }
 
+media-controller:is([media-paused], :not([user-inactive]))::part(vertical-layer) {
```

Comment:
target the vertical-layer part, where the controls are, when either the media is paused (and controls are showing) or when we're playing and the user is active and thus the controls are showing.

Reply:
I think we need `centered-layer` too. See: https://github.com/muxinc/media-chrome/blob/main/src/js/media-container.js#L118-L120

Reply:
nevermind, this should be fine! (we don't have a convention on which to use just yet)

Reply:
We want this to be as far behind as possible and for now, which should be the vertical layer. Plus, we're not letting mux-player customize controls yet, so, we should end up with both.